commands/search.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../lib/const'
+require_relative '../lib/package'
+require_relative '../lib/package_utils'
+
+class Command
+  def self.search(regex_string, verbose)
```

Comment:
Why not just abort right away if regex_string is empty instead of looping through every package only to output nothing?

Reply:
docopt handles that for us:
![image](https://github.com/chromebrew/chromebrew/assets/64618338/57125490-b261-4642-8ebe-61318f898bcb)

Reply:
Fair enough.  I'm sold.